components/avatar/index.css

```diff
@@ -10,107 +10,126 @@ OF ANY KIND, either express or implied. See the License for the specific languag
 }
 
 .spectrum-Avatar--size300 {
-  @remapvars {
-    find: /--spectrum-avatar-size-300(.*)/;
-    filter: color;
-    replace: --spectrum-avatar$1;
-  }
+  --spectrum-avatar-inline-size: var(--spectrum-avatar-size-300);
+  --spectrum-avatar-block-size: var(--spectrum-avatar-size-300)
 }
 
 .spectrum-Avatar--size400 {
-  @remapvars {
-    find: /--spectrum-avatar-size-400(.*)/;
-    filter: color;
-    replace: --spectrum-avatar$1;
-  }
+  --spectrum-avatar-inline-size: var(--spectrum-avatar-size-400);
+  --spectrum-avatar-block-size: var(--spectrum-avatar-size-400)
 }
 
 .spectrum-Avatar--size500 {
-  @remapvars {
-    find: /--spectrum-avatar-size-500(.*)/;
-    filter: color;
-    replace: --spectrum-avatar$1;
-  }
+  --spectrum-avatar-inline-size: var(--spectrum-avatar-size-500);
+  --spectrum-avatar-block-size: var(--spectrum-avatar-size-500)
 }
 
 .spectrum-Avatar--size600 {
-  @remapvars {
-    find: /--spectrum-avatar-size-600(.*)/;
-    filter: color;
-    replace: --spectrum-avatar$1;
-  }
+  --spectrum-avatar-inline-size: var(--spectrum-avatar-size-600);
+  --spectrum-avatar-block-size: var(--spectrum-avatar-size-600)
 }
 
 .spectrum-Avatar--size700 {
-  @remapvars {
-    find: /--spectrum-avatar-size-700(.*)/;
-    filter: color;
-    replace: --spectrum-avatar$1;
+  --spectrum-avatar-inline-size: var(--spectrum-avatar-size-700);
+  --spectrum-avatar-block-size: var(--spectrum-avatar-size-700)
+}
+
+@media (forced-colors: active) {
+  .spectrum-Avatar {
+    --highcontrast-avatar-color-opacity-disabled: 1;
+    --highcontrast-avatar-focus-indicator-color: Highlight;
   }
 }
 
 .spectrum-Avatar {
   display: inline-block;
   position: relative;
-  inline-size: var(--spectrum-avatar-width);
-  block-size: var(--spectrum-avatar-height);
+  inline-size: var(--mod-avatar-inline-size, var(--spectrum-avatar-inline-size));
+  block-size: var(--mod-avatar-block-size, var(--spectrum-avatar-block-size));
 
-  border-radius: var(--spectrum-avatar-border-radius);
+  border-radius: var(--mod-avatar-border-radius, var(--spectrum-avatar-border-radius));
   border-width: 0;
 
+  outline: none;
+
   -webkit-user-drag: none;
   user-select: none;
-  overflow: hidden;
 
-  &:after {
-    content: '';
+  overflow: visible;
+
+  opacity: var(--mod-avatar-color-opacity, var(--spectrum-avatar-color-opacity));
+
+  &.is-disabled {
+    opacity: var(--highcontrast-avatar-color-opacity-disabled, var(--mod-avatar-color-opacity-disabled, var(--spectrum-avatar-color-opacity-disabled)));
+  }
+
+  &:focus-within:not(.is-disabled) .spectrum-Avatar-link::after {
+    pointer-events: none;
+    content: "";
     position: absolute;
-    inset-block-start: 0;
-    inset-block-end: 0;
-    inset-inline-start: 0;
-    inset-inline-end: 0;
-    box-sizing: border-box;
-    border-width: var(--spectrum-avatar-border-size);
+    top: calc(
+      var(--mod-avatar-focus-indicator-gap, var(--spectrum-avatar-focus-indicator-gap)) * -2);
+    left: calc(
```

Comment:
This should be

```
    left: calc((var(--mod-avatar-focus-indicator-gap, var(--spectrum-avatar-focus-indicator-gap)) 
              + var(--mod-avatar-focus-indicator-thickness, var(--spectrum-avatar-focus-indicator-thickness))) * -1);
```


To see the difference, try changing line 21 to a larger pixel size, like

`--spectrum-avatar-focus-indicator-gap: 6px;`

Before the change, this resulted in an off-center focus indicator:
![Screenshot 2023-01-03 at 5 23 58 PM](https://user-images.githubusercontent.com/25614178/210457855-43420251-699d-44ea-bb76-e849f8c4e776.png)

After changing to the suggested calculation, the indicator centers even when the gap is a different amount than the border width:

![Screenshot 2023-01-03 at 5 24 24 PM](https://user-images.githubusercontent.com/25614178/210457884-b214f987-e8d0-4fc0-8cc5-fd71b6fdbf29.png)

Also, I think it _might_ be better to use margin-inline-start instead of "left".

Reply:
> Also, I think it might be better to use margin-inline-start instead of "left".

Today I learned! Similar to the comment about `top`, I think this is `inset-inline-start`: https://developer.mozilla.org/en-US/docs/Web/CSS/inset-inline-start